src/inverseRobot.js

```diff
@@ -24,7 +24,15 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
-}
+  const res = {};
```

Comment:
Let's create more meaningful variable name instead of res

Reply:
got it ty